pkg/simple/client/elasticsearch/esclient.go

```diff
@@ -30,28 +28,31 @@ import (
 var jsonIter = jsoniter.ConfigCompatibleWithStandardLibrary
 
 var (
-	mu        sync.RWMutex
-	esConfigs *ESConfigs
-)
-
-type ESConfigs struct {
-	Host  string
-	Port  string
-	Index string
-}
+	mu     sync.Mutex
```

Comment:
Why use Mutex instead of RWMutex?

Reply:
Because now we use elastic client, no need to frequently read config for structing http request as we once did: [L573](https://github.com/kubesphere/kubesphere/pull/656/files#diff-28320d9ea55aa094d44988b9161bbe6aL573)

Reply:
index is still read frequently. will fix and resubmitted